### ISSUE TITLE
'Forgot Password' and 'Register' too close

### ISSUE THREAD

**Issue by jotoeri** [nextcloud/server#26349]
### Actual Behaviour
On this picture from nextcloud/registration#276, i realised these buttons (or their order) are a bit confusing....
Just seeing this, it **looks to me like one should press the register-button for forgotten passwords.**
Any thoughts on this? @jancborchardt @nextcloud/designers 

### Expected behaviour
Non-confusing order 😉 

### Steps to reproduce
1. Have Registration activated
2. Open Login page

![grafik](https://user-images.githubusercontent.com/47433654/112747077-5bfa9980-8fb3-11eb-8b65-7945b65216f9.png)

**Comment by nimisha-vijay** [comment]
Yeah, I agree. It does look like you need to register if you forgot your password.
The order can be changed, or it can just be a spacing thing. If "Login" and "Forgot Password" are closer together and "Register" is farther down, then I think that's already an improvement.

**Comment by bes1002t** [comment]
imho the "Forgot Password?" text should be under the password input. Maybe we could group the login and register buttons and create a margin between them and the "Forgot Password?".

Like this example:
![nextcloud_login](https://user-images.githubusercontent.com/3890580/113515852-38b58880-9577-11eb-996a-7e97a45252be.png)

**Comment by jotoeri** [comment]
Hmm, i think i'd leave the space between Register and login, but having the Text under the input sounds reasonable to me. However, i'm not 'a Designer' so its probably just my single cent... 🙈

**Comment by nickvergessen** [comment]
![Screenshot_2021-04-06-12-59-12-761_com nextcloud talk2](https://user-images.githubusercontent.com/213943/113701179-17b87900-96d8-11eb-922d-0615cbb5d309.jpg)

Get's more confusing with "Log in with a device" option in 21+

**Comment by bes1002t** [comment]
@nickvergessen in this case I would suggest simply using my proposal and move the text "Log in with a device" directly above the login button.

**Comment by nickvergessen** [comment]
Related PR: https://github.com/nextcloud/server/pull/26586

**Comment by Ade-otoki** [comment]
@jotoeri The UI does look confusing and could be improved  by having a registration page open and a caption with "Already have account? Login" and the "forgot Password" would be on the Login.

I can help redesigning this, I quite new to open source so not quite sure how to begin

**Comment by joshtrichards** [comment]
Related: #45310
